agent/app/run.go

```diff
@@ -24,6 +26,10 @@ import (
 	log "github.com/cihub/seelog"
 )
 
+const (
```

Comment:
Question: Should we define and maintain all environment variables in the [config.go](https://github.com/aws/amazon-ecs-agent/blob/master/agent/config/config.go#L498) ? or this does not apply to healthcheck workflow? Thanks!

Reply:
i had wondered the same, see https://github.com/aws/amazon-ecs-agent/pull/2834#discussion_r597138412

Reply:
https://github.com/aws/amazon-ecs-agent/blob/a8f4fcb8be4e08830cbfb51096a403f2f263235e/agent/logger/log.go#L28
I see this more like the logger -- we don't use the config (and its variables) in the logger either.